extmod/network_wiznet5k.c

```diff
@@ -199,6 +200,10 @@ static void wiznet5k_config_interrupt(bool enabled) {
 }
 
 void wiznet5k_deinit(void) {
+    #if LWIP_MDNS_RESPONDER
+    mdns_resp_remove_netif(&wiznet5k_obj.netif);
```

Comment:
I think this should be moved down to just before the `netif_remove(netif)` line.

Reply:
Thanks for the review, @dpgeorge. I moved it as requested.